untitled/main/com/aca/homework/week11/occurence/KeyCalculator.java

```diff
@@ -0,0 +1,30 @@
+package com.aca.homework.week11.occurence;
+
+import java.util.*;
+
+public class KeyCalculator {
+    private Supplier stringOfNumberSupplier;
+
+    public KeyCalculator(Supplier stringOfNumberSupplier) {
+        this.stringOfNumberSupplier = stringOfNumberSupplier;
+    }
+
+    public Map<String, Integer> askAndCalculate() {
+        Map<String, Integer> map = new HashMap<>();
+        String inputStringNumber = (String) stringOfNumberSupplier.get();
```

Comment:
there is no need to cast

Reply:
but if  i don't cast ,the program does not compile saying : 
required type : String
perovided : Object
incompatible types: java.lang.Object cannot be converted to java.lang.String